### PR TITLE
Add infrastructure sponsor tier and add vercel to that category

### PR DESCRIPTION
**Pull request by IceSentry**
It uses the same size as the bronze tier, but uses a white color for the tier name.

image

You can preview the changes here [dev-bevy-webiste.icesentry.net](https://dev-bevy-webiste.icesentry.net/). Yes this page is hosted by vercel.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -223,6 +223,15 @@ <h2 class="sponsors__title">Bronze Sponsors</h2>
                 </a>
             </div>
         </div>
+        <div class="sponsors sponsors--infrastructure">
+            <h2 class="sponsors__title">Infrastructure Sponsors</h2>
+            <div class="sponsors__content">
+                <a class="sponsors__link" href="https://vercel.com">
```
Comment (alice-i-cecile) [inline templates/index.html:229]:
```suggestion
                <a class="sponsors__link" href="https://vercel.com?utm_source=[team-name]&utm_campaign=oss">
```

Per https://vercel.com/support/articles/can-vercel-sponsor-my-open-source-project

**Review by alice-i-cecile — APPROVED** [review]
This LGTM now. I'll pass it along to Vercel's team for approval.

File: templates/index.html [hunk 2]
```diff
@@ -223,6 +223,15 @@ <h2 class="sponsors__title">Bronze Sponsors</h2>
                 </a>
             </div>
         </div>
+        <div class="sponsors sponsors--infrastructure">
+            <h2 class="sponsors__title">Infrastructure Sponsors</h2>
+            <div class="sponsors__content">
+                <a class="sponsors__link" href="https://vercel.com?utm_source=[team-name]&utm_campaign=oss">
```
Comment (alice-i-cecile) [inline templates/index.html:229]:
```suggestion
                <a class="sponsors__link" href="https://vercel.com?utm_source=[bevy]&utm_campaign=oss">
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
 2. Clone the Bevy Website git repo and enter that directory:
     1. `git clone https://github.com/bevyengine/bevy-website.git`
     2. `cd bevy-website`
 3. Start the Zola server with `zola serve`.
 
 A local server should start and you should be able to access a local version of the website from there.
+
+## Infrastructure Sponsors
+
+We use Vercel to deploy preview branches to a live website to make development easier. <https://vercel.com>
```
Comment (alice-i-cecile) [inline README.md:21]:
```suggestion
We use [Vercel](https://vercel.com?utm_source=[bevy]&utm_campaign=oss) to deploy preview branches to a live website to make development easier.
```

**Comment by IceSentry** [comment]
It's hosted here if you want to send them a link instead https://dev-bevy-webiste.icesentry.net

**Comment by alice-i-cecile** [comment]
Part of #313.

File: sass/components/_sponsors.scss [hunk 4]
```diff
@@ -44,6 +44,7 @@
         ('tier': 'platinum', 'color': $color-white, 'height': 100px),
         ('tier': 'silver', 'color': #bbb, 'height': 80px),
         ('tier': 'bronze', 'color': #9f7d6c, 'height': 60px),
+        ('tier': 'infrastructure', 'color': $color-white, 'height': 60px),
```
Comment (doup) [inline sass/components/_sponsors.scss:47]: What do you think making the color a darker grey? "Iron grey". ^_^

```suggestion
        ('tier': 'infrastructure', 'color': #777, 'height': 60px),
```

File: templates/index.html [hunk 5]
```diff
@@ -223,6 +223,15 @@ <h2 class="sponsors__title">Bronze Sponsors</h2>
                 </a>
             </div>
         </div>
+        <div class="sponsors sponsors--infrastructure">
+            <h2 class="sponsors__title">Infrastructure Sponsors</h2>
+            <div class="sponsors__content">
+                <a class="sponsors__link" href="https://vercel.com?utm_source=[bevy]&utm_campaign=oss">
+                    <img class="sponsors__logo" src="assets/vercel/logotype/light/vercel-logotype-light.png"
```
Comment (doup) [inline templates/index.html:230]: Given that a SVG is available, I think it's better to use that, it weights less and gives better quality.
Also, what about putting it on a `sponsors` folder? We could move the other ones there also (in this PR, on another one).

```suggestion
                    <img class="sponsors__logo" src="assets/sponsors/vercel.svg"
```

Comment (doup) [reply templates/index.html:230]: Related to this, I would remove all other Vercel logos/icons.

File: sass/components/_sponsors.scss (hunk 4 above)
Comment (doup) [reply sass/components/_sponsors.scss:47]: This is how it looks like:
image

**Review by doup — APPROVED** [review]

File: templates/index.html [hunk 6]
```diff
@@ -223,6 +223,15 @@ <h2 class="sponsors__title">Bronze Sponsors</h2>
                 </a>
             </div>
         </div>
+        <div class="sponsors sponsors--infrastructure">
+            <h2 class="sponsors__title">Infrastructure Sponsors</h2>
+            <div class="sponsors__content">
+                <a class="sponsors__link" href="https://vercel.com?utm_source=[bevy]&utm_campaign=oss">
```
Comment (domeccleston) [inline templates/index.html:229]: This just needs to say `https://vercel.com?utm_source=bevy&utm_campaign=oss` (no square brackets required for the utm parameter)

File: templates/index.html (hunk 5 above)
Comment (IceSentry) [reply templates/index.html:230]: Yeah, good point, I just threw everything in there and didn't think much of it. I'll remove it. I'll also create a sponsors folder for all those assets to make it a bit more organized.

File: templates/index.html (hunk 6 above)
Comment (alice-i-cecile) [reply templates/index.html:229]: Ah, thanks! There's a couple places where we should correct this.

**Comment by alice-i-cecile** [comment]
Closing due to never getting said approval.

**Pull request closed without merging**